src/OrchardCore.Modules/OrchardCore.Spatial/GrapgQL/GeoPointFieldProvider.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using GraphQL.Resolvers;
+using GraphQL.Types;
+using Microsoft.Extensions.Localization;
+using OrchardCore.ContentManagement;
+using OrchardCore.ContentManagement.GraphQL.Queries.Types;
+using OrchardCore.ContentManagement.Metadata.Models;
+using OrchardCore.Spatial.Fields;
+
+namespace OrchardCore.Spatial.GrapgQL;
+public class GeoPointFieldProvider : IContentFieldProvider
+{
+    protected readonly IStringLocalizer S;
+    public GeoPointFieldProvider(IStringLocalizer<GeoPointFieldProvider> stringLocalizer)
+    {
+        S = stringLocalizer;
+    }
+
+    private static object handleFieldResolving(ContentElement field)
```

Comment:
```suggestion
    private static object HandleFieldResolving(ContentElement field)
```

Please use PascalCase.